api/views.py

```diff
@@ -169,4 +177,87 @@ def enrichment_view(request):
         data=request.query_params, context={"request": request}
     )
     serializer.is_valid(raise_exception=True)
+
+    source_ip = str(request.META["REMOTE_ADDR"])
+    request_source = Statistics(source=source_ip, view=viewType.ENRICHMENT_VIEW.value)
+    request_source.save()
+
     return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class StatisticsViewSet(viewsets.ViewSet):
```

Comment:
With empty data four graph (the exception is Feeds: Types) show an error message with status code 500.
I suggest to add some log